docs/core/diagnostics/well-known-event-providers.md

```diff
@@ -0,0 +1,110 @@
+
+## Framework Libaries
+
+### "Microsoft-Extensions-DependencyInjection" provider
+
+This provider logs information from DependencyInjection. The table below shows events logged by `Microsoft-Extensions-DependencyInjection` provider:
+
+|Event name|Level|Description|
+|----------|-----|-----------|
+|`CallSiteBuilt`|Verbose (5)|A call site has been built.|
+|`ServiceResolved`|Verbose (5)|A service has been resolved.|
+|`ExpressionTreeGenerated`|Verbose (5)|An expression tree has been generated.|
+|`DynamicMethodBuilt`|Verbose (5)|A <xref:System.Reflection.Emit.DynamicMethod> has been built.|
+
+### "System.Buffers.ArrayPoolEventSource" provider
+
+This provider logs information from the ArrayPool. The table below shows the events logged by `ArrayPoolEventSource`:
+
+|Event name|Level|Description|
+|----------|-----|-----------|
+|`BufferRented`|Verbose (5)|A buffer is successfully rented.|
+|`BufferAllocated`|Informational (4)|A buffer is allocated by the pool.|
+|`BufferReturned`|Verbose (5)|A buffer is returned to the pool.|
+|`BufferTrimmed`|Informational (4)|A buffer is attempted to be freed due to memory pressure or inactivity.|
+|`BufferTrimPoll`|Informational (4)|A check is being made to trim buffers.|
+
+### "System.Net.Http" provider
+
+This provider logs information from the HTTP stack. The table below shows the events logged by `System.Net.Http` provider:
+
+|Event name|Level|Description|
+|----------|-----|-----------|
+|RequestStart|Informational (4)|An HTTP request has started.|
+|RequestStop|Informational (4)|An HTTP request has finished.|
+|RequestFailed|Error (2)|An HTTP request has failed.|
+|ConnectionEstablished|Informational (4)|An HTTP connection has been established.|
+|ConnectionClosed|Informational (4)|An HTTP connection has been closed.|
+|RequestLeftQueue|Informational (4)|An HTTP request has left the request queue.|
+|RequestHeadersStart|Informational (4)|An HTTP request for header has started.|
+|RequestHeaderStop|Informational (4)|An HTTP requset for header has finished.|
+|RequestContentStart|Informational (4)|An HTTP request for content has started.|
+|RequestContentStop|Informational (4)|An HTTP request for content has finished.|
+|ResponseHeadersStart|Informational (4)|An HTTP response for header has started.|
+|ResponseHeaderStop|Informational (4)|An HTTP response for header has finished.|
+|ResponseContentStart|Informational (4)|An HTTP response for content has started.|
+|ResponseContentStop|Informational (4)|An HTTP response for content has finished.|
+
+### "System.Net.NameResolution" provider
+
+This provider logs information related to domain name resolution (DNS). The table below shows the events logged by `System.Net.NameResolution`:
```

Comment:
DNS isn't really the abbreviation for domain name resolution. Maybe leave it out or do this?

```suggestion
This provider logs information related to DNS domain-name resolution. The table below shows the events logged by `System.Net.NameResolution`:
```

Reply:
Since DNS stands for domain name resolution, wouldn't it sound a little weird to say DNS domain-name resolution?